feathr_project/feathr/api/app/core/configs.py

```diff
@@ -1,12 +1,12 @@
 import logging
 import os
 
-API_Version = "/v1"
+API_Version = "/api/v1"
 log_level = os.getenv("logLevel", "INFO")
 app_insights_connection_string = os.getenv("APPLICATIONINSIGHTS_CONNECTION_STRING")
 formatter = logging.Formatter("[%(asctime)s] [%(name)s:%(lineno)s - %(funcName)5s()] %(levelname)s - %(message)s")
 appServiceKey =  "mockAppServiceKey" # os.getenv("AppServiceKey")
-
+default_purview_name = "feathrazuretest3-purview1"
```

Comment:
Why are we hardcoding the name here?

Reply:
I will work with @blrchen on adding this to env, and read from env in later updates, thanks for reminding